tests/etc/responses.yml.example

```diff
@@ -104,6 +104,15 @@ alcatel_aos:
   version_banner: "Alcatel-Lucent OS6250-24 6.7.1.108.R04 Service Release, January 04, 2017"
   multiple_line_output: "FC - ForcedCopper PC - PreferredCopper C - Copper"
 
+
+netscaler:
+  base_prompt: ""
```

Comment:
@ktbyers this required a test-change--not the ideal "fix", but I'm not sure it's applicable for these devices